gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -66,11 +72,17 @@ def __init__(self,
                test_name,
                table_name,
                input_pattern,
+               validation_query,
```

Comment:
I'd prefer these to be a list of tuples so that you can run multiple queries with verification for each query on the same table. You can add a TODO for now though.

Reply:
Updated the TODO on line 151 which is for enhancing queries.